homeassistant/components/rpi_i2c_expanders.py

```diff
@@ -0,0 +1,98 @@
+"""
+Support for controlling I²C port expanders connected to Raspberry Pi
+
+"""
+# TODO:  For more details about this component,
+# please refer to the documentation at
+# https://home-assistant.io/components/TODO/
+
+import logging
+
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_START,
+    EVENT_HOMEASSISTANT_STOP,
+)
+
+REQUIREMENTS = ["smbus-cffi==0.5.1"]
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "rpi_i2c_expanders"
+
+g_managed_chips = None
+g_pollwatcher = None
+
+
+def setup(hass, config):
+    """Set up RPi i²C expanders component."""
+    import smbus
+    from . import rpi_i2c_ha_expanders
+
+    global g_managed_chips
```

Comment:
Don't use global. `hass.data` is available to store data that needs to be shared between eg component and platform.